tracer/src/Datadog.FleetInstaller/Commands/RemoveGlobalInstrumentation.cs

```diff
@@ -0,0 +1,67 @@
+// <copyright file="RemoveGlobalInstrumentation.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+using System.Threading.Tasks;
+
+namespace Datadog.FleetInstaller.Commands;
+
+/// <summary>
+/// Remove IIS instrumentation completely
+/// </summary>
+internal class RemoveGlobalInstrumentation : CommandBase
```

Comment:
the most nitpicky of nitpicks but:
```suggestion
internal class RemoveGlobalInstrumentationCommand: CommandBase
```

Reply:
Would have to also update the XML docs and ctor if accepted so not really important

Reply:
However I see other classes missing Command so 🤷

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/7110/commits/ea1a542d5ed94382dcd794b397125763763091ae